### PR TITLE
Add custom "about" window

### LINKED ISSUE #511: Link to privacy statement/policy in about
https://github.com/wulkano/kap/pull/492#issuecomment-409608352
 
https://github.com/wulkano/kap/blob/master/PRIVACY.md

### LINKED ISSUE #524: Improve about window focus
**macOS Version:** 10.13.6
**Kap Version:** 3.0.0-beta.1.1006

#### Steps to reproduce
- Open Kap (default recording interface)
- Click the ... icon and then "About Kap Beta"

#### Current behaviour
- The About window is opened
- It is not focused or interact-able because the recording interface takes mouse focus
- Cannot close or interact with the About window until you exit the recording interface

#### Expected behaviour
- Exit the recording interface
- Open the About window
- Basically just use the same behaviour as the Preferences window

#### Screenshot
<p align="center">
screen shot 2018-09-24 at 11 36 20 am
</p>

It feels natural that a newly created window should receive focus and is not hidden behind the recording interface.

### PR DESCRIPTION
**Pull request by karaggeorge**
Fixes #511 
Fixes #524

### CODE REVIEW

**Comment by sindresorhus** [comment]
Why are we creating a custom about window? Why not just use the native one? Seems like an easier solution to the focus issue would be to just close the cropper view when the about window is opened.

**Comment by sindresorhus** [comment]
The about window should be slightly wider to exactly match the default macOS about window.

**Comment by sindresorhus** [comment]
I think we should also include a `Website` link in there, on the left side of the Policy link.

**Comment by sindresorhus** [comment]
The icon should not be draggable. Use `pointer-event: none` on it.

**Comment by karaggeorge** [comment]
Is there a way to execute code when the about window is opened? It seemed like it was an accelerator so there was no way to close the croppers before opening it.

File: main/about.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+const openAboutWindow = () => {
+  closeAllCroppers();
+
+  if (aboutWindow) {
+    aboutWindow.show();
+    return;
+  }
+
+  aboutWindow = new BrowserWindow({
+    width: 280,
```
Comment (sindresorhus) [inline main/about.js]: Native about window is `284`

**Comment by sindresorhus** [comment]
I didn't think of that. If it's an accelerator, then no.

**Comment by sindresorhus** [comment]
The about window should not show until it's completely rendered. This is what I see in the built app (not dev):

kap-about 2018-09-27 19_57_57

**Comment by karaggeorge** [comment]
Not sure why that happened. I called `show` on `componentDidUpdate`, but as you said I still noticed the flash. Awaiting `delay(100)` seems to solve it.

**Pull request merged**